lib/block_view.go

```diff
@@ -7039,11 +7142,253 @@ func (bav *UtxoView) _connectAcceptNFTBid(
 			len(txMeta.UnlockableText), bav.Params.MaxPrivateMessageLengthBytes)
 	}
 
-	// Get the poster's profile.
+	// Verify the NFT bid entry being accepted exists and has a bid consistent with the metadata.
+	// If we did not require an AcceptNFTBid txn to have a bid amount, it would leave the door
+	// open for an attack where someone replaces a high bid with a low bid after the owner accepts.
+	nftBidKey := MakeNFTBidKey(txMeta.BidderPKID, txMeta.NFTPostHash, txMeta.SerialNumber)
```

Comment:
Am I missing something or is this redundant? It seems like you're doing this exact same check in the _helpConnectNFTSold function. So maybe you can delete this from here?

Reply:
yes it is redundant! deleting.